src/pages/ProjectInsights.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="project-insights">
+    <div class="container project-insights__container">
+      <b-row class="align-items-stretch">
+        <b-col v-for="(widget, index) in instantiatedWidgets" :key="index" :md="widget.cols">
+          <div class="project-insights__container__widget" :class="{ card: widget.card }">
+            <component :is="widget.component" :widget="widget" class="flex-grow-1" />
+          </div>
+        </b-col>
+      </b-row>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+
+import ProjectSelector from '@/components/ProjectSelector'
+import utils from '@/mixins/utils'
+
+export default {
+  components: {
+    ProjectSelector
+  },
+  mixins: [utils],
+  props: {
+    name: {
+      type: String
+    }
+  },
+  computed: {
+    ...mapGetters('insights', ['instantiatedWidgets'])
+  },
+  watch: {
+    name(name) {
+      this.$store.commit('insights/project', this.name)
+    }
+  },
+  beforeMount() {
+    this.$store.commit('insights/project', this.name)
```

Comment:
It might be easier to retrieve the project name from the URL maybe

Reply:
`this.name` is a prop directly [imported from the route](https://github.com/ICIJ/datashare-client/blob/46eadbf15bab9f6de92cbffdacd2d309c28aa2f2/src/router/index.js#L211).

Reply:
Oops sorry I missed that